tensorflow-core/tensorflow-core-api/pom.xml

```diff
@@ -328,6 +328,36 @@
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
```

Comment:
That's perfect @rnett , thanks. I'll suggest that you add a comment what this plugin/goal is doing, like the other goals in the same `pom.xml`, it helps to know what is going on since there are a lot of steps involved. 

We can do that when we'll uncomment the phase as well.